src/mlpack/methods/ann/layer/advantage_loss.hpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * @file advantage_loss.hpp
```

Comment:
Do you think we should name the file ``advantage_error.hpp`` the same a the class name?

Reply:
I feel that naming it advantage error is to be too specific. It represents class of policy gradient approach , hence I believe we should name it as  policygradient.hpp . This will make it generalised . 

I named it particularly as advantage_error for my luxury to implement the algorithm. However , we can always change it.